greenpithumb/greenpithumb.py

```diff
@@ -47,35 +49,29 @@ def __init__(self, wiring_config):
             self._dht11)
         self._humidity_sensor = humidity_sensor.HumiditySensor(self._dht11)
 
-    def print_readings_header(self):
-        """Print a header for sensor values to be printed in columns."""
-        print 'light\tmoisture\ttemperature\thumidity'
-
-    def print_readings(self):
-        """Print sensor values in columns."""
-        light_reading = '%.1f%%' % self._light_sensor.ambient_light()
-        moisture_reading = '%.1f' % self._moisture_sensor.moisture()
-        temperature_reading = (('%.1f' % self._temperature_sensor.temperature())
-                               + u'\N{DEGREE SIGN} C')
-        humidity_reading = '%.1f' % self._humidity_sensor.humidity()
-        print '%s\t%s\t\t%s\t\t%s' % (light_reading, moisture_reading,
-                                      temperature_reading, humidity_reading)
-
 
 def read_wiring_config(config_filename):
     with open(config_filename) as config_file:
         return wiring_config_parser.parse(config_file.read())
 
 
+def create_record_processor(db_connection, record_queue):
```

Comment:
Yeah, I've thought about it, but I couldn't think of a good reason to underscore them. It's sort of like in tests where it just feels like extra keystrokes for no value.